### PR TITLE
Update license files for adblock lists

### PR DESCRIPTION
**Pull request by antonok-edm**
Resolves https://github.com/brave/brave-browser/issues/11003

This change includes the licenses for the uBlock Origin lists we will be switching to.

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/third_party/adblock/README.chromium [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Name: Brave Ad Block
-URL: https://github.com/brave/brave-core-crx-packager
```
Comment (pes10k) [inline components/third_party/adblock/README.chromium:2]: This name (adblock) was a reference to the C++ adblock library.  Should this whole section / file be updated to point to adblock-rs?

File: components/third_party/adblock/lists/peter_lowes/LICENSE [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Preamble
```
Comment (pes10k) [inline components/third_party/adblock/lists/peter_lowes/LICENSE:1]: Lets not use this, lets use Peter's statement :)

``` I'm Peter Lowe, and Brave Software has permission to use my list in
 its software.

 pgl@yoyo.org
```

File: components/third_party/adblock/README.chromium (hunk 1 above)
Comment (antonok-edm) [reply components/third_party/adblock/README.chromium:2]: It previously pointed to the CRX packager with the license set to "various", so I figured it is referring to our collection of filter lists as a whole - which I think `adblock-resources` is most representative of for now

Comment (pes10k) [reply components/third_party/adblock/README.chromium:2]: okie dokie :)

File: components/third_party/adblock/lists/peter_lowes/LICENSE [hunk 3]
```diff
@@ -0,0 +1,3 @@
+its software.
```
Comment (fmarier) [inline components/third_party/adblock/lists/peter_lowes/LICENSE:1]: Is there something missing here?

Comment (antonok-edm) [reply components/third_party/adblock/lists/peter_lowes/LICENSE:1]: Perhaps I pressed "resolve" too quickly! :sweat_smile: 
https://github.com/brave/brave-core/pull/6256#discussion_r462595437

**Comment by antonok-edm** [comment]
Test failure is from rewards and is unrelated, merging

**Pull request merged**